File: WIS/source/WIS_Prototype/WIS_Prototype/src/app/report/producer-report.ts
export class ProducerReport {
  producerName: string;
  districtName: string;
  warehouseName: string;
  commodityTypeName: string;
  commodityVarietyName: string;
  netWeightLbs: number;
  sumLoads: number;

  constructor() {
    this.producerName = '';
    this.districtName = '';
    this.warehouseName = '';
    this.commodityTypeName = '';
    this.commodityVarietyName = '';
    this.netWeightLbs = 0;
    this.sumLoads = 0;
  }
}

File: WIS/source/WIS_Prototype/WIS_PrototypeAPI/Data/DTOs/ProducerReport.cs
using WIS_PrototypeAPI.Data.Models;

namespace WIS_PrototypeAPI.Data.DTOs
{
	public class ProducerReport
	{

		public string ProducerName {get; set;}
		public string DistrictName { get; set; }

		public string WarehouseName { get; set; }

		public string CommodityTypeName { get; set; }

		public string CommodityVarietyName { get; set; }

		public int NetWeightLbs { get; set; }

		public int SumLoads { get; set; }
	}
}
